### ISSUE TITLE
Skipping processing BackupSession. Invalid character '\x00'.

### ISSUE THREAD

**Issue by akirill0v** [stashed/stash#1040]
I'm using Postgresql addon for Stash (v0.9.0-rc.4).
Snapshot created, but BackupSession finished with Error:

```
Status:
  Phase:  Failed
  Targets:
    Phase:  Failed
    Ref:
      Kind:  AppBinding
      Name:  potok-postgres
    Stats:
      Error:      invalid character '\x00' looking for beginning of value
```

Container with name `postgres-backup-11-2-0` finish logs:

```
{"message_type":"status","seconds_elapsed":1081,"seconds_remaining":342113248050,"percent_done":0,"total_files":1,"bytes_done":58288284958,"current_files":["/dumpfile.sql"]}
{"message_type":"status","seconds_elapsed":1081,"seconds_remaining":342113248050,"percent_done":0,"total_files":1,"files_done":1,"bytes_done":58288284958}
{"message_type":"status","seconds_elapsed":1082,"seconds_remaining":342425189539,"percent_done":0,"total_files":1,"files_done":1,"bytes_done":58288284958}
{"message_type":"status","seconds_elapsed":1082,"seconds_remaining":342425189539,"percent_done":0,"total_files":1,"files_done":1,"bytes_done":58288284958}
{"message_type":"summary","files_new":0,"files_changed":1,"files_unmodified":0,"dirs_new":0,"dirs_changed":0,"dirs_unmodified":0,"data_blobs":50,"tree_blobs":1,"data_added":150400379,"total_files_processed":1,"total_bytes_processed":0,"total_duration":1082.057377248,"snapshot_id":"0efb45b0"}
```

and status container logs:

```
I0128 01:18:40.597948       1 status.go:56] Reading backup output from file: /tmp/output/output.json
I0128 01:18:40.634760       1 status.go:104] Updating status of BackupSession: postgres/daily-postgres-backup-1580173223 for host: host-0
I0128 01:18:40.698873       1 main.go:43] Exiting Stash Main
```

**Comment by hossainemruz** [comment]
Can you provide full logs from `postgres-backup-11-2-0` container?

**Comment by akirill0v** [comment]
Hello, @hossainemruz. I'll make gist with full log. Strange symbols in the end of log.
I maked log with command
`k -n postgres logs -f stash-backup-daily-postgres-backup-1580173223-0-42s8j -c postgres-backup-11-2-0 > postgres-backup-11-2-0 .txt`

https://gist.github.com/akirill0v/613cc5588f76003d1eaf83542a8d649f

**Comment by DerFetzer** [comment]
I am having the same issue but while backing up a StatefulSet. After the last status message restic outputs a number of `0x00` Bytes.
Could be related to this restic [issue](https://github.com/restic/restic/issues/2389#issuecomment-572641462). The [fix](https://github.com/restic/restic/pull/2545) has already been merged to master but is not included in `0.9.6` release.

**Comment by aarononeal** [comment]
When will the fix for this be merged? I have a few backups a day reporting failed because of it.
